actions/user/registration/socialSignUp.js

```diff
@@ -0,0 +1,13 @@
+export default function socialSignUp(context, payload, done) {
+    context.service.create('user.socialregistration', payload, { timeout: 20 * 1000 }, (err, res) => {
+        if (err) {
+            context.dispatch('SOCIAL_CREATE_USER_FAILURE', err);
+            done();
```

Comment:
According to [Fluxible Action Docs](https://github.com/yahoo/fluxible/blob/master/packages/fluxible/docs/api/Actions.md) you could omit the `done()` call in case you're using promises. Also have a look at this [Fluxible Actions Docs](http://fluxible.io/api/actions.html) for correct usage of promises with dispatch.
The following should work:
```
function socialSignUp(context, payload) {
....
  return context.dispatch(....);
....
}
```